src/components/cms/BrowseSkill/BrowseSkill.js

```diff
@@ -677,15 +677,25 @@ class BrowseSkill extends React.Component {
     } else if (searchQuery.length > 0) {
       noSkillFound = (
         <NoSkillFound>
-          <h2 style={{ fontWeight: '400' }}>
-            Your search <b>&quot;{searchQuery}&quot;</b> did not match any
-            skills.
-          </h2>
-          <h3 style={{ margin: '15px 0 10px 0' }}>Try something like</h3>
-          <ul style={{ listStyle: 'inside' }}>
-            <li>Using more general terms</li>
-            <li>Checking your spelling</li>
-          </ul>
+          <div style={{ display: 'flex', flexDirection: 'column' }}>
+            <h2 style={{ fontWeight: '400' }}>
+              Your search <b>&quot;{searchQuery}&quot;</b> did not match any
+              skills.
+            </h2>
+            <div
+              style={{
```

Comment:
Use styled-components for styling

Reply:
ok, are you referring to the div(line 680) that I have created and want these changes using styled-components.

Reply:
@paras151 Yes